src/widget/OktaSignIn.js

```diff
@@ -185,7 +181,8 @@ var OktaSignIn = (function () {
       },
       token: {
         hasTokensInUrl: hasTokensInUrl,
-        parseTokensFromUrl: parseTokensFromUrl
+        parseTokensFromUrl: parseTokensFromUrl,
+        renew: renewToken
```

Comment:
mostly because is the new https://github.com/okta/okta-signin-widget/pull/641/files/ab080ea55d7094fe5770f3c1fc1ff9207853c371#diff-6f46e9fe9976e77793ac9317de08f44aL178 which was exposed, but I am ok removing it if not necessary

Reply:
IMO, we should standardize on a pattern since we're introducing a breaking change.

Since `token.renew` accepts then token as a parameter, and `tokenManager.renew` accepts the token key, I'd suggest keeping the `tokenManager` method as it is a bit easier to call.

Reply:
Agree to keep `tokenManager.renew` between the twos, will remove `token.renew`. 👍